src/main/resources/config/modules/Vanilla.xml

```diff
@@ -726,9 +726,9 @@
             <BiomeType name='Swamp'  />
             <BiomeType name='Beach'  />
             <BiomeType name='River'  />
-            <Biome name='Ocean'  weight='-1' />
-            <Biome name='Desert'  weight='-1' />
-            <Biome name='Mountain'  weight='-1' />
+            <Biome name='.*Ocean.*'  weight='-1' />
```

Comment:
I'm pretty sure that if a distribution specifies any biome restriction, none of the restrictions from the parent are inherited. The same goes for ore blocks and replacement. Just settings are inherited.